src/gdcmodels/es-models/case_centric/vestigial.yaml

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
All these vestigial files are in preparation to be populated in another step? I may have missed it, but I didn't see any new data for these.

Reply:
These will be populated for the mappings that require them in the next step